core/controllers/base.py

```diff
@@ -605,11 +638,13 @@ def render_template(self, filepath, iframe_restriction='DENY'):
 
         self.response.write(load_template(filepath))
 
-    def _render_exception_json_or_html(self, return_type, values):
+    def _render_exception_json_or_html(
+        self, return_type: Optional[str], values: ResponseValueDict
+    ) -> None:
         """Renders an error page, or an error JSON response.
 
         Args:
-            return_type: str. Indicator to return JSON or HTML.
+            return_type: str|None. Indicator to return JSON or HTML.
```

Comment:
This seems wrong, shouldn't it be a non-optional str? What does None represent conceptually?

Reply:
Done !

Reply:
Erm, so, are you planning to remove the None possibility in this method then?

Reply:
Docstring updated!